<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryViewDialog</class>
 <widget class="QDialog" name="BinaryViewDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="DisplayModeWidget" name="displayModeWidget" native="true"/>
   </item>
   <item>
    <widget class="BinaryMap" name="binaryMap" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1,0,0">
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="lblCtrl">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Ctrl: Modify Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblShift">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Shift: Modify End</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QLabel" name="lblIncrement">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>+/- Offset Increment:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SpinBoxBaseSelector" name="sbStep" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cursor Offset:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCursorOffset">
       <property name="text">
        <string>00000000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBoxBaseSelector</class>
   <extends>QWidget</extends>
   <header>spinboxbaseselector/spinboxbaseselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BinaryMap</class>
   <extends>QWidget</extends>
   <header>views/hexview/binaryviewdialog/binarymapwidget/binarymap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DisplayModeWidget</class>
   <extends>QWidget</extends>
   <header>views/hexview/binaryviewdialog/displaymodewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
